controllers/dexserver_controller.go

```diff
@@ -478,6 +478,15 @@ func (r *DexServerReconciler) syncDeployment(dexServer *authv1alpha1.DexServer,
 	// Iterate over connectors defined in the DexServer to create the dex configuration for connectors
 	for _, connector := range dexServer.Spec.Connectors {
 		if connector.Type == authv1alpha1.ConnectorTypeLDAP && connector.LDAP.RootCARef.Name != "" {
+
+			// Check if secret is in the dex server namespace
```

Comment:
Do we need this check since we already handled in syncConfigMap?

Reply:
@robinbobbitt I guess this may be unnecessary. I kept it in on the off chance that the secret did not finish getting copied over into the dex server ns. But I think I can pull it out